src/main/webapp/js/instructorFeedbackEdit/questionMsq.es6

```diff
@@ -0,0 +1,80 @@
+/* global
+FEEDBACK_QUESTION_NUMBEROFCHOICECREATED:false, FEEDBACK_QUESTION_MSQCHOICE:false
+*/
+
+function addMsqOption(questionNum) {
+    const questionId = `#form_editquestion-${questionNum}`;
+
+    const curNumberOfChoiceCreated =
+            parseInt($(`#${FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNum}`).val(), 10);
```

Comment:
Just a random comment, but maybe sometime in the future we may want to put a constant somewhere like `const RADIX_DECIMAL = 10`, just so that this doesn't look like a magic number.